modules/apps/collaboration/social/social-user-statistics-web/src/main/resources/META-INF/resources/add_activity_counter.jsp

```diff
@@ -23,6 +23,10 @@ String portletResourceNamespace = PortalUtil.getPortletNamespace(portletResource
 
 int index = ParamUtil.getInteger(request, "index");
 
+if (index == 0) {
+	index = 1;
+}
+
```

Comment:
Do we need this block?  When I was testing without this block of code, it looks like the fields increment correctly, only that it goes 0, 2,3,4...instead of 1,2,3,4.

Reply:
I only added it because incrementing 0,2,3,4 seems strange to me and could be perceived as a bug.

Reply:
My follow up to that then is why is it jumping 1?

Reply:
As discussed, I only bring this up because on https://github.com/jonmak08/liferay-portal/pull/1807/files/3fb7c744a965549773e5cc25fde06596c06b0d30#diff-66bb28b0c416b176cf4dcc14bc7cbee7R36 we do a check for the ```index < displayActivityCounterNames.length``` and by manually incrementing it to 1 this check should not pass.